source/MaterialXRender/Handlers/HwLightHandler.h

```diff
@@ -0,0 +1,111 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_HWLIGHTHANDLER_H
+#define MATERIALX_HWLIGHTHANDLER_H
+
+/// @file
+/// Handler for hardware lights
+
+#include <MaterialXCore/Definition.h>
+#include <MaterialXCore/Node.h>
+
+#include <string>
+#include <memory>
+
+namespace MaterialX
+{
+class GenContext;
+
+/// Shared pointer to a LightHandler
+using HwLightHandlerPtr = std::shared_ptr<class HwLightHandler>;
+
+/// @class HwLightHandler
+/// Utility light handler for creating and providing
+/// light data for shader binding.
+///
+class HwLightHandler
```

Comment:
Maybe we should rename this class and file to just LightHandler, for consistency, since no other handler has the Hw prefix.